Snowflake.Data.Tests/IntegrationTests/SFDbCommandIT.cs

```diff
@@ -175,6 +175,379 @@ public void TestExecuteAsyncWithMaxRetryReached()
                 Assert.GreaterOrEqual(stopwatch.ElapsedMilliseconds, 30 * 1000);
             }
         }
+
```

Comment:
it would be great to have a test showing how big problem would be revealing a queryId...
The scenario to test would be that in one connection we create an async long running query and in another one for different user/role/schema we try to get the result for given queryId. It would be good if we weren't allowed to get the result.

Reply:
I tested this using two different accounts:
 - On the original account, the result was retrieved using the query id
 - On a different account, the query status was `NO_DATA` using the same query id even when the query already finished executing in the original account